include/LightGBM/config.h

```diff
@@ -806,6 +807,12 @@ struct Config {
   // desc = when ``multi_error_top_k=1`` this is equivalent to the usual multi-error metric
   int multi_error_top_k = 1;
 
+  // desc = file containing matrix of weights for calculating AUC-mu multi-class metric
+  // desc = matrix must be square, with number of rows and columns equal to number of classes
+  // desc = if not specified, will use equal weights for all classes
+  // desc = see `this file <https://github.com/microsoft/LightGBM/tree/master/examples/multiclass_classification/loss.weights>`__ as an example
+  std::string auc_mu_weights_file = "";
```

Comment:
python/R package should pass a file as well?
Could we have another hyper-parameter, maybe called `auc_mu_weights`, whose size is n*n. And you can use `auc_mu_weights[i * c + j]` to get the `auc_mu_weights[i][j]`. We have many other hyper-parameters which are array format too, like `label_gain`.
Then, it will be easier to use in python/R package. And maybe `auc_mu_weights_file` could be removed too.

Reply:
Thanks, I made this change and removed the parameter `auc_mu_weights_file`.